documentation/source/testbench_tools.rst

```diff
@@ -1,218 +0,0 @@
-***************
-Testbench Tools
-***************
-
-Logging
-=======
-
-Cocotb uses the builtin :mod:`logging` library, with some configuration described in :ref:`logging-reference-section` to provide some sensible defaults.
-Each :term:`DUT`, monitor, driver, and scoreboard (as well as any other function using the coroutine decorator) holds a :class:`logging.Logger`, and each can be set to its own logging level.
-Within a :term:`DUT`, each hierarchical object can also have individual logging levels set.
-
-When logging :term:`HDL` objects, beware that ``_log`` is the preferred way to use
-logging. This helps minimize the change of name collisions with an :term:`HDL` log
-component with the Python logging functionality.
-
-Log printing levels can also be set on a per-object basis.
```

Comment:
I'm not sure if we have this information somewhere else still, maybe some of it could move to "Writing Testbenches"?

Reply:
I had thought about it, but I would need to rewrite the examples to not reference the bus objects. I took the lazy way out. I'll work on porting it tonight.

Reply:
I wasn't thinking of the examples really (I could live without them), just this information about `logging` being used and that each object can have its own log level etc.

Reply:
So just move the highlight bit to "Writing Testbenches" in the logging section? I can do that.

Reply:
This was moved to Writing Testbenches and the wording and examples were touched up.